Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_8.yml

```diff
@@ -1274,6 +1283,66 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    scriptarguments:
+      Account:
+        complex:
+          root: Account
+          transformers:
+          - operator: uniq
+      CriticalEndpoints:
+        simple: admin
+      CriticalGroups:
+        simple: admins,administrators
+      CriticalUsers:
+        simple: admin,administrator
```

Comment:
why this is being passed manually ? and not as an input or thorough the parent PB inputs?

Reply:
Same as previous comment